### PR TITLE
Feat/detect related files

### PR DESCRIPTION
**Pull request by LuisTejedaS**
## Description

Resolve related files from an array of WSDL root files

## How to test this ticket
 npm run test

### CODE REVIEW

File: lib/SchemaPack.js [hunk 1]
```diff
@@ -214,6 +217,47 @@ class SchemaPack {
     }
   }
 
+  /**
+   *
+   * @description Takes in a folder and identifies the related files from the
+   * root file perspective (using $ref property)
```
Comment (erfemega) [inline lib/SchemaPack.js:223]: Should we change using $ref property from description?

**Review by Carlos-Veloz — APPROVED** [review]

File: lib/relatedFiles.js [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+      else {
+        missingNodes.push({ schemaLocation: calculatedPathForMissing.schemaLocation, path: null });
+      }
+    }
+  });
+  return { graphAdj, missingNodes };
+}
+
+/**
+   * Maps the output from get root files to detect root files
```
Comment (erfemega) [inline lib/relatedFiles.js:149]: Is this description correct?

Comment (LuisTejedaS) [reply lib/relatedFiles.js]: Changed

File: lib/SchemaPack.js (hunk 1 above)
Comment (LuisTejedaS) [reply lib/SchemaPack.js]: changed

**Review by erfemega — APPROVED** [review]

**Pull request merged**